django_socio_grpc/protobuf/generation_plugin.py

```diff
@@ -316,3 +331,69 @@ def transform_request_message(
                 service, proto_message, message_name_constructor
             )
         return proto_message
+
+
+@dataclass
```

Comment:
dataclass is not needed as you don't use any attribute.
Could have been usefull if use a param for enumPlugin instead of multiple class but both are good.

Reply:
you resolved without a comment ad modification

Reply:
sry removed all but this one, done now